src/zac/elasticsearch/tests/utils.py

```diff
@@ -15,7 +20,17 @@ def refresh_index(self):
         zaken = Index(settings.ES_INDEX_ZAKEN)
         zaken.refresh()
 
+    def create_zaak_document(self, zaak):
+        if not isinstance(zaak, Zaak):
+            zaak = factory(Zaak, zaak)
+        create_zaak_document(zaak)
+
+    def add_rol_to_document(self, rol):
+        if not isinstance(rol, Rol):
+            rol = factory(Rol, rol)
+        append_rol_to_document(rol)
+
     def setUp(self):
         super().setUp()
 
-        self._clear_index()
+        self.clear_index()
```

Comment:
slightly more robust:

```suggestion
        self.addCleanup(self.clear_index)
```